cunumeric/module.py

```diff
@@ -2498,6 +2499,140 @@ def take(
     return a.take(indices=indices, axis=axis, out=out, mode=mode)
 
 
+def _fill_fancy_index_for_along_axis_routines(
+    a_shape: tuple, axis: int, indices: ndarray
+) -> tuple:
```

Comment:
Also needs full type spec, `tuple[<something>]` (or `NdShape`  if that is applicable)

Reply:
thanks! I am still learning how to correctly do type annotation. It should be tuple of ndarrays